app/model/editions/templates/TemplateHelpers.scala

```diff
@@ -81,26 +82,66 @@ object SpecialEditionButtonStyles {
   implicit val formatSpecialEditionButtonStyles : OFormat[SpecialEditionButtonStyles] = Json.format[SpecialEditionButtonStyles]
 }
 
-trait EditionDefinition {
+/**
+  * A curated product that has a push publication model – it
+  * pushes updated content to the world in discrete 'editions' that contain a
+  * list of fronts.
+  *
+  * Contrast with e.g. web/app Fronts, which publish their content in real time
+  * on a container by container basis.
+  */
+trait CuratedPlatform {
   val title: String
   val subTitle: String
   val edition: String
-  val header: Header
-  val editionType: EditionType
   val notificationUTCOffset: Int
-  val topic: String
   val locale: Option[String]
+  val app: String
```

Comment:
Maybe this should be an enum - what do you think?

Reply:
Agreed! 51ee9274dddf750f8e3d958b58b33cea1b720df9